.circleci/config.yml

```diff
@@ -43,10 +43,10 @@ commands:
       - restore_cache:
           name: Restoring Composer Cache
           keys:
-              - composer-v1-{{ checksum "composer.json" }}-<< parameters.php-image >>-<< parameters.guzzle-version >>
-              - composer-v1-{{ checksum "composer.json" }}-<< parameters.php-image >>
-              - composer-v1-{{ checksum "composer.json" }}
-              - composer-v1-
+              - composer-v3-{{ checksum "composer.json" }}-<< parameters.php-image >>-<< parameters.guzzle-version >>
```

Comment:
What is the reason for renaming v1 -> v3 here ?

Reply:
The current cache is not valid for new image.